docs/gallery.yml

```diff
@@ -1,6 +1,6 @@
-- title: Face Area Calculations
-  path: examples/003-area-calc.ipynb
-  thumbnail: _static/thumbnails/default.svg
+# - title: Face Area Calculations
+#   path: examples/003-area-calc.ipynb
+#   thumbnail: _static/thumbnails/default.svg
```

Comment:
We can remove these comments and delete the original notebook from the examples directory.